src/contexts/wishlist-context.js

```diff
@@ -0,0 +1,21 @@
+import {createContext, useContext, useReducer} from 'react'
+import {wishlistReducer} from "../reducers/wishlist-reducer"
+
+const wishContext = createContext()
+
+const WishlistProvider = ({children}) => {
+    const [wishState, wishDispatch] = useReducer(wishlistReducer, {
+        wishlistProducts : [],
+        wishlisted : false
```

Comment:
Hey,

The code looks great, just one suggestion
I don't think there is a need to use wishlisted (Boolean) in your state object.
All objects inside your wishlistProducts array will anways mean that they have been wishlisted.

Rest looks good to merge